scripts/link-data.sh

```diff
@@ -80,6 +81,11 @@ link_locs=(
   scRNA-seq-advanced/data/PBMC-TotalSeqB/normalized/PBMC_TotalSeqB_normalized_sce.rds
   scRNA-seq-advanced/data/glioblastoma/raw_feature_bc_matrix
   scRNA-seq-advanced/data/reference
+  # only the _integration_ rms files
+  scRNA-seq-advanced/data/rms/processed/SCPCL000478.rds
+  scRNA-seq-advanced/data/rms/processed/SCPCL000479.rds
+  scRNA-seq-advanced/data/rms/processed/SCPCL000480.rds
+  scRNA-seq-advanced/data/rms/processed/SCPCL000481.rds
```

Comment:
I'm debating whether including all of the files might be the right move here. We would have to change the instruction notebook to specify the files to integrate in that case though, so I'm fine either way.

Reply:
Yup, it's related to my file parsing choices in that notebook. We can revisit this later as needed!